### DISCUSSION TITLE
Dịch streaming metric và stateful metric

### DISCUSSION THREAD

**Discussion by hnguyentt** [mlbvn/handson-ml2-vn#125] [Terminology] [answered]
Chương 12 có các thuật ngữ: streaming metric | stateful metric trong context:
>After the first batch, it returns a precision of 80%; then after the second batch, it returns 50% (which is the overall precision so far, not the second batch’s precision). This is called a **streaming metric** (or **stateful metric**), as it is gradually updated, batch after batch.

Mình đề xuất dịch:
* streaming metric: phép đo trực tuyến
* stateful metric: phép đo trạng thái

**Comment by minhduc0711** [comment, marked as answer]
* streaming metric: em thấy `phép đo luồng` có vẻ sát nghĩa hơn
* stateful metric: `phép đo **có** trạng thái`

**Comment by minhduc0711** [comment]
ping nhẹ @mlbvn/handson-ml
